client/my-sites/site-settings/custom-content-types/index.jsx

```diff
@@ -52,6 +53,13 @@ class CustomContentTypes extends Component {
 		return isRequestingSettings || isSavingSettings;
 	}
 
+	requestSitePostTypes = () => {
+		const { siteId } = this.props;
+		if ( siteId ) {
+			this.props.requestPostTypes( siteId );
```

Comment:
See https://github.com/Automattic/wp-calypso/pull/14576/files#diff-5832a88e66f1c149f5666b93306f55e3R176 - since we're passing an object to `mapDispatchToProps`, actions are bound automatically to `dispatch`.